test/integration/cli-image-create-kvm.test.js

```diff
@@ -30,13 +29,13 @@ var IMAGE_DATA = {
     version: '1.0.0'
 };
 var DERIVED_INST_ALIAS = _RESOURCE_NAME_PREFIX + '-derived';
-delete _RESOURCE_NAME_PREFIX;
 
 var testOpts = {
     skip: (!h.CONFIG.allowWriteActions || h.CONFIG.skipKvmTests) &&
         'requires config.allowWriteActions and config.skipKvmTests=false'
 };
 
+_RESOURCE_NAME_PREFIX = null;
```

Comment:
Nit - why even bother nulling this out (or even deleting it) - can't it just be left as it was originally defined?

Reply:
I'd say so, but I'm unaware of the reason why it was added. I'll just do as you suggest, since it seems to be the right thing for me to do too. Let's see if it can surprise us or was just some fancy stuff here. So far it was present on the file since the day it was added to git.